rdt/hyper_transformer.py

```diff
@@ -831,7 +719,7 @@ def create_anonymized_columns(self, num_rows, column_names):
 
         transformers = []
         for column_name in column_names:
-            transformer = self._transformers_tree.get(column_name, {}).get('transformer')
+            transformer = self.field_transformers.get(column_name, {})
```

Comment:
I don't think we should return a dictionary as a backup here. The returned value should be a transformer, so if it doesn't exist shouldn't we let it be `None`?